show/muxcable.py

```diff
@@ -0,0 +1,339 @@
+import json
+import sys
+
+import click
+import utilities_common.cli as clicommon
+from sonic_py_common import multi_asic
+from swsscommon import swsscommon
+from swsssdk import ConfigDBConnector
+from tabulate import tabulate
+
+from utilities_common import platform_sfputil_helper
```

Comment:
Group third-party imports together and alphabetize:

```python
import click
import utilities_common.cli as clicommon
from sonic_py_common import multi_asic
from swsscommon import swsscommon
from swsssdk import ConfigDBConnector
from tabulate import tabulate
from utilities_common import platform_sfputil_helper
```

Reply:
fixed